Cargo.toml

```diff
@@ -18,8 +18,11 @@ default = []
 serde = ["dep:serde", "egui/serde"]
 
 [dependencies]
-egui = "0.20"
+egui = { version = "0.21", default-features = false }
```

Comment:
Small change here: egui has the `default_fonts` feature by default. It bundles the default egui fonts (hence the name).

Previously, `egui_dock` would implicitly "force" the `default_fonts` feature onto users of `egui_dock`, bloating their binaries if they were not using `default_fonts` themselves.

Reply:
Pardon my cargo-ignorance; if a user wanted to use the version of `egui` exported by `egui_dock`, what would they need to do to enable `default_fonts` in it?